### ISSUE TITLE
do you use the pre_trained glove feature?

### ISSUE THREAD

**Issue by TAY-985** [JonghwanMun/LGI4temporalgrounding#16]
Hello, I see glove_path is none.  Didn't you use glove during training?
![image](https://user-images.githubusercontent.com/65696081/140631021-e058598e-96f4-4885-a051-78ca55ee5383.png)

**Comment by JonghwanMun** [comment]
Yes, I **do not** use GLOVE features when initializing word vectors.

**Comment by TAY-985** [comment]
okay,thanks . but how do you initialize word features?

**Comment by JonghwanMun** [comment]
I apply the random initialization scheme, that is, training word features from scratch.

**Comment by TAY-985** [comment]
thank you
